inspirehep/base/static/js/search/search_results.js

```diff
@@ -157,7 +157,9 @@ define(
         var format_id = '#' + $(ev.target).attr("id");
         var obj = this.get_download_information($(format_id).text().trim());
         $("#response-data").html('');
-        $("#dropdown-export").html('BibTex <span class="caret"></span>')
+        $("#dropdown-export").html('BibTex <span class="caret"></span>');
+        $("#copy-to-clipboard").addClass('disabled');
```

Comment:
Are all these things related to the copy clipboard needed on the search results but not on the detailed page?